server/controllers/responseFormatters.js

```diff
@@ -1,7 +1,11 @@
 const { pick } = require('./utils')
 
-const formatLeagueResp = (leagueDoc) =>
-    pick(leagueDoc, ['_id', 'name', 'organisation', 'creator', 'admins', 'seasons'])
+const formatLeagueResp = (leagueDoc) => {
+    const creator = pick(leagueDoc.creator, ['_id', 'email', 'firstName', 'lastName'])
```

Comment:
Not 100% sure but you might need to edit the tests because now the creator object will be getting more details in its response and not just the id.

Reply:
ok will do